analytics.go

```diff
@@ -0,0 +1,78 @@
+package flagsmith
+
+import (
+	"context"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+)
+
+const AnalyticsTimerInMilli = 10 * 1000
```

Comment:
I guess this is `DefaultAnalyticsTimerInMilli` or can you not configure the analytics timer? 🤔

Reply:
No, you can't configure analytics timer